python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -241,6 +241,15 @@ def popMjdDate(self, md, key, timesys=None):
         return BadDate
 
     @staticmethod
+    def eraFromLstAndLongitude(lst, longitude):
```

Comment:
Also I believe this takes lsst.afw.geom.Angle objects, NOT angles in degrees. Please fix the documentation.